rem.go

```diff
@@ -29,14 +29,21 @@ Options:
    -t/--set-dir <dir>     set the data dir and continue
    -q/--quiet             enable quiet mode
    --disable-copy         if files are on a different fs, don't rename by copy
+   -f/--force             Do not print error message on inexistant file,
+                          used for compatibility with rm
    -h/--help              print this help message
-   -v/--version           print Rem version`
+   --version              print Rem version
+   -i/--interactive       ignored, used for compatibility with rm
+   -r/-R/--recursive      ignored, used for compatibility with rm
+   -v/--verbose           ignored, used for compatibility with rm
+   --                     all arguments after this as considered files`
```

Comment:
Let's make a note after all the options that says "Rem ignores flags used by `rm` for compatibility. Use `--` when files resemble flags."

We don't need to mention each flag that's ignored.